src/sort/i64.rs

```diff
@@ -54,6 +54,11 @@ impl Sort for I64Sort {
         add_primitives!(typeinfo, "<=" = |a: i64, b: i64| -> Opt { (a <= b).then(|| ()) });
         add_primitives!(typeinfo, ">=" = |a: i64, b: i64| -> Opt { (a >= b).then(|| ()) });
 
+        add_primitives!(typeinfo, "bool-<" = |a: i64, b: i64| -> bool { a < b });
```

Comment:
An alternative set of names could be `is-less`, `is-greater`, `is-not-greater`, and `is-not-less` which makes it clear that they return booleans.

Reply:
Okay that sounds better

Reply:
Hmm, actually I like this for now because it tips you off on the bool output